Pixel Game/Assets/Scripts/PlayerScripts/PlayerInventory.cs

```diff
@@ -271,7 +186,7 @@ public void LoadEquipmentOnStartScene()
             if (!slot.IsEmpty && dataToPass.savedEquipmentListDB.Count != 0 && 
                 dataToPass.savedEquipmentListDB.Contains(slot.ItemDataInSlot.itemIdString))
             {
-                slot.EquipItem();
+                StartCoroutine(slot.EquipItem());

```

Comment:
Why do we use Coroutine for eqipitem on scene change?

Reply:
It's because of a change I made that enables use of projectile weapons bought from store and/or carried between scenes. I activate the gameobjects for one frame with `yield return null` so that they can be properly initialized.
Forgot to mention this in PR description.